js/search/searchConfig.js

```diff
@@ -26,12 +26,15 @@ const libraryPath = isMac ? path.join(macLibraryPath, 'libsymphonysearch.dylib')
 const userConfigFileName = 'search_users_config.json';
 const userConfigFile = isDevEnv ? path.join(__dirname, '..', '..', userConfigFileName) : path.join(userData, userConfigFileName);
 
+const freeDiskSpace = path.join(winLibraryPath, 'free-disk-space.exe');
```

Comment:
i see we are checking in a binary file.
why do we need it?  what does it do?
where are we getting this binary file from?
do we trust it?
assume only runs on windows???
will it run on all windows platforms?  does it have DLL dependencies?

Reply:
@lneir 
1. Before I was using the `fsutils` Which requires admin rights to execute (details in this [link](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/fsutil))
2. Here is the [link](https://msdn.microsoft.com/en-us/library/system.io.driveinfo.availablefreespace(v=vs.110).aspx?cs-save-lang=1&cs-lang=csharp#code-snippet-2) for checking the `AvailableFreeSpace` code in C#.
I'm using this code and made some changes as per required and compiled using `csc` [link](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/command-line-building-with-csc-exe).
3. And the binary only check for free disk space for the provided drive name.
4. As it does not require any dependency and only using system.io which works on all windows platform.

Reply:
where is the source code?  we should have this in a repo and have builds setup to provide exe.